downstream/modules/aap-hardening/con-credential-management-planning.adoc

```diff
@@ -7,9 +7,10 @@
 
 [role="_abstract"]
 
-{ControllerNameStart} uses credentials to authenticate requests to jobs against machines, synchronize with inventory sources, and import project content from a version control system. {ControllerNameStart} manages three sets of secrets:
+{PlatformName} uses credentials to authenticate requests to jobs against machines, synchronize with inventory sources, and import project content from a version control system. {ControllerNameStart} manages three sets of secrets:
 
-* User passwords for *local automation controller users*. See the xref:con-user-authentication-planning_{context}[User Authentication Planning] section of this guide for additional details.
+* User passwords for *local automation controller users*. 
+//See the xref:con-user-authentication-planning_{context}[User Authentication Planning] section of this guide for additional details.
 * Secrets for automation controller *operational use* (database password, message bus password, and so on).
```

Comment:
Consider using the attribute for automation controller in this line {ControllerName}